### PR TITLE
Set suppress_filters to false when adding a WooCommerce widget

### PR DESCRIPTION
**Pull request by adrianosferreira**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6209

### CODE REVIEW

**Comment by adrianosferreira** [comment]
@OnTheGoSystems/wpml please review.

**Comment by adrianosferreira** [comment]
@OnTheGoSystems/wpml hold on, I made a mistake in this PR. I will fix and ask again for review.

**Comment by adrianosferreira** [comment]
@OnTheGoSystems/wpml now it is ready, please review.

File: src/class-wpml-elementor-woocommerce-hooks-factory.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+class WPML_Elementor_WooCommerce_Hooks_Factory implements IWPML_Backend_Action_Loader {
```
Comment (strategio) [inline src/class-wpml-elementor-woocommerce-hooks-factory.php:3]: Don't we need it also on the frontend @adrianosferreira?

Comment (adrianosferreira) [reply src/class-wpml-elementor-woocommerce-hooks-factory.php:3]: @strategio for now no, it is only needed when we are adding new widgets. It happens on Elementor editor which runs on WP admin scope.

**Review by strategio — APPROVED** [review]
+1 @adrianosferreira

**Pull request merged**